File: 03-conditional-statements-advanced/exercises-submitted-evaluated/12.tradeCommissions.js
function tradeCommissions(input) {
    let city = input[0];
    let s = Number(input[1]);
    let commission = 0;
    switch (city) {
        case ("Sofia"):
            if (s >= 0 && s <= 500) {
                commission = 0.05 * s;
                console.log(commission.toFixed(2));
            } else if (s > 500 && s <= 1000) {
                commission = 0.07 * s;
                console.log(commission.toFixed(2));
            } else if (s > 1000 && s <= 10000) {
                commission = 0.08 * s;
                console.log(commission.toFixed(2));
            } else if (s > 10000) {
                commission = 0.12 * s;
                console.log(commission.toFixed(2));
            } else {
                console.log("error");
            }
            break;
        case ("Varna"):
            if (s >= 0 && s <= 500) {
                commission = 0.045 * s;
                console.log(commission.toFixed(2));
            } else if (s > 500 && s <= 1000) {
                commission = 0.075 * s;
                console.log(commission.toFixed(2));
            } else if (s > 1000 && s <= 10000) {
                commission = 0.1 * s;
                console.log(commission.toFixed(2));
            } else if (s > 10000) {
                commission = 0.13 * s;
                console.log(commission.toFixed(2));
            } else {
                console.log("error");
            }
            break;
        case ("Plovdiv"):
            if (s >= 0 && s <= 500) {
                commission = 0.055 * s;
                console.log(commission.toFixed(2));
            } else if (s > 500 && s <= 1000) {
                commission = 0.08 * s;
                console.log(commission.toFixed(2));
            } else if (s > 1000 && s <= 10000) {
                commission = 0.12 * s;
                console.log(commission.toFixed(2));
            } else if (s > 10000) {
                commission = 0.145 * s;
                console.log(commission.toFixed(2));
            } else {
                console.log("error");
            }
            break;
        default:
            console.log("error");
            break;
    }
}
tradeCommissions(["Sofia", "1500"]);
tradeCommissions(["Plovdiv", "499.99"]);
tradeCommissions(["Varna", "3874.50"]);
tradeCommissions(["Kaspichan", "-50"]);

File: softuni-python-basics/03_conditional_statements_advanced/12_trade_commissions.py
city = input()
sales = float(input())
commission_rate = 0
valid = True

if city == "Sofia":
    if 0 <= sales <= 500:
        commission_rate = 0.05
    elif 500 < sales <= 1000:
        commission_rate = 0.07
    elif 1000 < sales <= 10000:
        commission_rate = 0.08
    elif sales > 10000:
        commission_rate = 0.12
    else:
        valid = False
elif city == "Varna":
    if 0 <= sales <= 500:
        commission_rate = 0.045
    elif 500 < sales <= 1000:
        commission_rate = 0.075
    elif 1000 < sales <= 10000:
        commission_rate = 0.1
    elif sales > 10000:
        commission_rate = 0.13
    else:
        valid = False
elif city == "Plovdiv":
    if 0 <= sales <= 500:
        commission_rate = 0.055
    elif 500 < sales <= 1000:
        commission_rate = 0.08
    elif 1000 < sales <= 10000:
        commission_rate = 0.12
    elif sales > 10000:
        commission_rate = 0.145
    else:
        valid = False
else:
    valid = False

if not valid:
    print("error")
else:
    commission_value = sales * commission_rate
    print(f"{commission_value:.2f}")

